### DISCUSSION TITLE
No new tab

### DISCUSSION THREAD

**Discussion by EffeF** [infinitepower18/Homepage-MobileSafari#16] [Support] [answered]
When I close the last tab I get to the default Safari homepage, my homepage is only loaded when I open a new tab. Is this behavior normal? Can it also be replaced in the other case?

I seem to remember that before iOS 18 the default Safari homepage was never shown, and even by closing the last tab I was correctly redirected to the personalized homepage.

**Comment by infinitepower18** [comment]
Yeah this is out of my control unfortunately. I'll submit a bug report to Apple as this issue doesn't happen on older iOS versions.

**Reply by EffeF** [reply]
Damn, this is very annoying... I hope then that Apple will fix the issue soon. 

Thanks for the quick response!

**Comment by infinitepower18** [comment]
I reported the issue to Apple last month but so far haven’t got a response yet. It’s still an issue on the latest iOS 18.3 beta.

When I hear anything back, I will update the [known issues](https://github.com/infinitepower18/Homepage-MobileSafari/discussions/17) page.

**Comment by infinitepower18** [comment, marked as answer]
Hi, I have an update regarding this.

While it's still a problem on iOS 18.6, I'm happy to report that Apple has fixed the issue in iOS 26. If Apple doesn't fix this in a future iOS 18 update I suggest updating to iOS 26 once it's officially released sometime next month.
